packages/ui/src/composables/useSelectableList.ts

```diff
@@ -31,18 +27,18 @@ export function useSelectableList (props: ExtractPropTypes<typeof useSelectableL
   }
 
   const getValue = (option: SelectableOption) => getOptionProperty(option, props.valueBy)
+
   const getOptionByValue = (value: SelectableOption) => {
     if (!props.valueBy) { return value }
-
     return props.options.find((option: SelectableOption) => value === getValue(option)) || value
   }
+
   const getText = (option: SelectableOption): string => getOptionProperty(option, props.textBy)
-  const getDisabled = (option: SelectableOption): boolean => getOptionProperty(option, props.disabledBy)
+  const getDisabled = (option: SelectableOption): boolean => !isStringOrNumber(option) && getProp(option, props.disabledBy)
```

Comment:
Just a question. I don't really understand what is going on here.

Is it shorthand for `option !== string && option !== 0`?

Looks like getProp returns.... undefined whet option is string. Can this be handled in getProp?

```ts
  if (typeof option !== 'object') { return option }
  if (!prop || !option) { return option }
  if (typeof prop === 'string') { return getValueByPath(option, prop) }
  if (typeof prop === 'function') { return prop(option) }
  return option
```

Reply:
Updated `getProp` and renamed to `getValueByKey`